requirements.txt

```diff
@@ -23,3 +23,4 @@ pytest-mock==1.7.1
 django-postgres-stats==1.0.0
 https://github.com/datamade/pytest-flake8/zipball/master
 sentry-sdk==0.5.1
+django-cacheops==4.1
```

Comment:
does this interact at all with the django cache? is the flow like, a request is made, if there's a page in the cache, load that, otherwise execute the view code, using cached objects from cache-ops if they exist?

Reply:
(like i understand we've turned off django caching for the search and login pages, but will there be any side effects to the remaining pages?)